dll.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+from __future__ import unicode_literals
+
+
+class Node(object):
+    ''' Create data Node with default value and next pointer. '''
+    def __init__(self, value, ahead=None, behind=None):
+        ''' Value and next pointer default to none. '''
+        self.val = value
+        self.ahead = ahead
+        self.behind = behind
+
+
+class DLL(object):
+    ''' Create an empty DLL. '''
+    def __init__(self):
+        self.head = None
+        self.tail = None
+
+    def insert(self, value):
+        ''' Add data Node to the head of DLL. '''
+        self.head = Node(value, behind=self.head)
+
+        ''' Check tail value. '''
+        if not self.tail:
+            self.tail = self.head
+        else:
+            self.head.behind.ahead = self.head
+
+    def pop(self):
+        ''' Remove head Node from head of DLL. Reassign head data
+            Node and return value of popped Node. '''
+        try:
+            val = self.head.val
+        except AttributeError:
+            ''' Mimic error message from list '''
+            raise IndexError("pop from empty DLL")
+        self.head.behind.ahead, self.head = None, self.head.behind
+        return val
+
+    def append(self, value):
+        ''' Add data Node to the tail of DLL. '''
+        self.tail = Node(value, self.tail)
```

Comment:
Just a little thing: if you're implementing ahead as a keyword argument, it's probably best to call it as a keyword argument. Otherwise, you're forced to keep all of your kwargs in their current positions. By using ahead = self.tail, you can add another positional argument to DLL without breaking this. Just something to think about.